db/migrate/20230512095159_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Let's follow the ERD on all the column names. A user should be named author in other tables. I advise you to do research about creating custom foreign keys and learn how a user will be named author in other tables.

Hint:
 ```ruby
    t.references :author, foreign_key: { to_table: 'users' } # This will create author_id and a corresponding index 
```

Reply:
Hi @elisha2kyakpo1

Thank you for your feedback
I already created another migration that renames the foreign key, so you can refer to 20230512100427_change_foreign_key_for_posts.rb in the migrate folder.
-Thank you!